.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "jenkins-cli"]
+  path = pkg/client/devops/jenkins-cli 
+  url = https://github.com/wxharry/jenkins-cli.git
+  branch = dev-ks-devops
+[submodule "pkg/client/devops/jenkins-cli"]
```

Comment:
Please let us know why this submodule is needed here.

Reply:
I use `jenkins-cli` because it is based on JSON requests to interact with Jenkins.
And I made some changes in `jenkins-cli`'s `dev-ks-devops` branch. I add a new function `createJobInFolder` which enables the client to create a job inside a folder.

Reply:
IMO, we just need first sub module. Please see more [here](https://git-scm.com/book/en/v2/Git-Tools-Submodules).

Reply:
Thanks! I found something useful in the link and update the submodule. Sorry for the delay.